src/baldr/admin.cc

```diff
@@ -4,58 +4,57 @@ namespace valhalla {
 namespace baldr {
 
 // Constructor given parameters.
-Admin::Admin(const uint32_t country_offset, const uint32_t state_offset,
-             const std::string& country_iso, const std::string& state_iso)
-    : country_offset_(country_offset), state_offset_(state_offset){
+Admin::Admin(const uint32_t country_offset,
+             const uint32_t state_offset,
+             const std::string &country_iso,
+             const std::string &state_iso)
+    : country_offset_(country_offset), state_offset_(state_offset) {
 
   std::size_t length = 0;
   // Example:  GB or US
   if (country_iso.size() == kCountryIso)
-    length = country_iso.copy(country_iso_,kCountryIso);
-  else country_iso_[0]= '\0';
+    length = country_iso.copy(country_iso_, kCountryIso);
+  else
+    country_iso_[0] = '\0';
 
   // Example:  PA
-  if (state_iso.size() == kStateIso-1) {
-    length = state_iso.copy(state_iso_,kStateIso-1);
-    state_iso_[length]='\0';
+  if (state_iso.size() == kStateIso - 1) {
+    length = state_iso.copy(state_iso_, kStateIso - 1);
+    state_iso_[length] = '\0';
   }
   // Example:  WLS
   else if (state_iso.size() == kStateIso)
-    length = state_iso.copy(state_iso_,kStateIso);
-  else state_iso_[0]= '\0';
+    length = state_iso.copy(state_iso_, kStateIso);
+  else
+    state_iso_[0] = '\0';
```

Comment:
@TheMarex Is there a way to enforce the use of brackets?

Reply:
Sadly I don't think this is possible with `clang-format` since it only transforms formatting. There is another tool called `clang-tidy` that is able to do this, but would need additional time to integrate.

Reply:
Now would be the time to introduce clang-tidy for this - it'll likely have a similar touch-many-things impact if we add it, we might as well do it all in one megacommit so that history is simpler to navigate.

I'm a strong 👍 on requiring braces around if statements - I've lost count of the number of bugs I've seen that were cause by the unclarity caused by clearly nesting blocks with `{}`.

Reply:
@danpat if you want to give it a go, I will be out for :beach_umbrella: next week. :wink: